[18.0][MIG] product_packaging_level_vendor: Migration to 18.0

thienvh332 opened the pull request:
### Context:
- `product_packaging_type` was renamed to `product_packaging_level`.
### Changes:
- Change `product_packaging_type_vendor` to `product_packaging_level_vendor`
### Depends on:
- `[#1806](https://github.com/OCA/product-attribute/pull/1806)`

Commit:
[ADD] product_sequence: Allows to associate a sequence to the product reference. The reference (default code) is unique (SQL constraint) and required.

sebastienbeau committed:
[MERGE] merge translation commited by launchpad on extra-addons6.0

bguillot committed:
[FIX] fix write all product with default_code together and then write each product without default_code

eLBati committed:
[FIX] resetting sequence code when updating module

Commit:
[REM] use Model instead of osv.osv

some minor reformatting in the process

sebastienbeau committed:
[FIX] product_sequence : write should support list or integer

Commit:
[MIGR] migration of product_sequence to openerp 7.0 (required for product_images)

Commit:
FIX description of module product_custom_attributes.
Version numbers should only have 2 digits.

Commit:
Launchpad automatic translations update.

Commit:
Set all addons to uninstallable and move them into __unported__ folder

Commit:
MIG product_sequence

Commit:
Added german translation

yvaucher committed:
Add bug tracker link on README.rst

gdgellatly committed:
product_sequence

Add pre_init_hook to manually set nulls and '/' default codes
 to a unique code to greenify branch
add pragma to pre_init method as it must have completed successfully
if module installs

yvaucher committed:
Add missing default oca icons

SodexisTeam committed:
product_sequence (#156)

atchuthan committed:
[MIG][10.0] migrate product_sequence

oca-transbot committed:
OCA Transbot updated translations from Transifex

LoisRForgeFlow committed:
[11.0][MIG/IMP] product_sequence:
* Migration to v11
* Support sequence by Product Category

JordiBForgeFlow committed:
fixes and adds tests

sbidoul committed:
remove obsolete .pot files [ci skip]

oca-travis committed:
[UPD] Update product_sequence.pot

sudhir-erpharbor committed:
[MIG] Migrated product_sequence to v12

Tardo committed:
[IMP] product_sequence: Suggested PR changes

oca-travis committed:
[UPD] Update product_sequence.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-12.0/product-attribute-12.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_sequence/

sergio-teruel committed:
[12.0][FIX] product_sequence: Keep default_code when copy products if dictionary values contains the default_code key.

OCA-git-bot committed:
product_sequence 12.0.1.0.1

sergio-teruel committed:
[IMP] product_sequence: black, isort

sergio-teruel committed:
[MIG] product_sequence: Migration to v13.0

oca-travis committed:
[UPD] Update product_sequence.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-13.0/product-attribute-13.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_sequence/

benbrich committed:
Translated using Weblate (German)

Currently translated at 100.0% (9 of 9 strings)

Translation: product-attribute-13.0/product-attribute-13.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_sequence/de/

OCA-git-bot committed:
pre-commit update

hparfr committed:
Add module product_logistics_uom

Commit:
[ADD] product_category_archive: new module

pedrobaeza committed:
[FIX] product_category_archive: Constraint search refinement

The problem why child_of was not working directly in the search is because there are
involved inactive records that are excluded by default by the ORM.

We need to disable that exlusion for a proper working, and it's needed because the
other domain is not taking into account possible inactive categories in the middle
of the hierarchy.

Commit:
[IMP] product_category_archive: add a test

Commit:
[ADD] product_medical: create

Commit:
[IMP] product_medical: Change in layout, add multiple docs

1. The layout for the old tab MDR (now called Conformity)
   has been changed to split the fields into two categories:
   conformity-related and documents-related.

2. The two fields storing the "Declaration of Conformity" and
   the "CE Certificate Medical class", that before stored an
   attachment, now allow to store several attachments.
       To make this change, a migration script has been
   provided, because the field names have changed to adhere
   to the naming conventions.

3. The string for the field ce_certificate_medical_class_ids
   has changed from 'CE Certificate Medical class' to 'CE Certificate'.
   The translation files have been updated.

didierdonze committed:
Translated using Weblate (German)

Currently translated at 68.3% (28 of 41 strings)

Translation: product-attribute-13.0/product-attribute-13.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_medical/de/

didierdonze committed:
Translated using Weblate (French)

Currently translated at 100.0% (9 of 9 strings)

Translation: product-attribute-13.0/product-attribute-13.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_sequence/fr/

MiquelRForgeFlow committed:
[UPD] Eficent -> ForgeFlow

OCA-git-bot committed:
[UPD] README.rst

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (17 of 17 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_logistics_uom/es/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

Translated using Weblate (Portuguese)

Currently translated at 50.0% (2 of 4 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_category_active/pt/

watthanun committed:
[MIG] product_sequence: Migration to 14.0

[UPD] Update product_sequence.pot

[UPD] README.rst

hparfr committed:
[MIG] product_logistics_uom: Migration to 14.0

hparfr committed:
[IMP] product_logistics_uom: black, isort, prettier

khaeusler committed:
Use parent categories to determine the prefix if the category has no settings for the prefix.

[UPD] Update product_sequence.pot

[UPD] README.rst

product_sequence 14.0.2.0.1

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-14.0/product-attribute-14.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_sequence/

Translated using Weblate (Dutch)

Currently translated at 100.0% (18 of 18 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_sequence/nl/

claudiagn committed:
Added translation using Weblate (Catalan)

Translated using Weblate (Catalan)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_category_active/ca/

CarlosRoca13 committed:
[IMP] product_category_active: black, isort, prettier

CarlosRoca13 committed:
[MIG] product_category_active: Migration to v13.0

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Added translation using Weblate (Catalan)

Commit:
[IMP] product_medical: views for conformity categories

The attributes for the Conformity Configuration (Medical Class,
Medical Category, PPE Category, and In Vitro Diagnostics) had no
views associated to them, so the only way of accessing them was
through the entry they have in the form for product.template.

With this PR, tree views and their actions have been added, so
that they can be reused in other modules to accessed them
from custom menu-items.

Commit:
[FIX] product_medical: minor changes in labels

- Medical Class -> Medical Device Class
- Medical Category > Drug Category

dzungtran89 committed:
[IMP] product_medical: black, isort, prettier

dzungtran89 committed:
[MIG] product_medical: Migration to 14.0

legalsylvain committed:
[ADD][12.0] new module product_net_weight.
- Add new net_weight field on product models;
- Rename weight field with 'Gross Weight' string;

OCA-git-bot committed:
[ADD] icon.png

rousseldenis committed:
[14.0][MIG] product_category_active

manuelregidor committed:
[IMP]product_net_weight: black, isort, prettier

manuelregidor committed:
[MIG]product_net_weight: Migration to 14.0

tgrellety committed:
[MIG] product_category_active: Migration to 15.0

BertVGroenendael committed:
[MIG] product_sequence: Migration to 15.0

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (52 of 52 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_medical/ca/

flachica committed:
[MIG][15.0] Standard migration product_net_weight

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
product_net_weight 15.0.1.0.1

flachica committed:
[MIG] product_logistics_uom: 15.0

marielejeune committed:
[MIG] product_sequence: Migration to 16.0

[UPD] Update product_sequence.pot

[UPD] README.rst

product_sequence 16.0.1.0.1

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/

Translated using Weblate (German)

Currently translated at 100.0% (16 of 16 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/de/

marielejeune committed:
[MIG] product_logistics_uom: Migration to 16.0

Commit:
[UPD] Update product_net_weight.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-15.0/product-attribute-15.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-15-0/product-attribute-15-0-product_net_weight/

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 76.9% (10 of 13 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_logistics_uom/de/

marielejeune committed:
[FIX] product_sequence unit tests: invalidate_cache, refresh and flush are deprecated

marielejeune committed:
[IMP] product_sequence: allow creating in batch for product.product and product.category

product_sequence 16.0.1.0.2

[UPD] Update product_sequence.pot

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/

victoralmau committed:
[FIX] product_net_weight: Fix digits parameter.

OCA-git-bot committed:
product_net_weight 15.0.1.0.2

rousseldenis committed:
[IMP] product_packaging_type: Rename to product_packaging_level

rousseldenis committed:
[IMP] product_packaging_level: Add migration scripts

carlosdauden committed:
Added translation using Weblate (Spanish)

carlosdauden committed:
Translated using Weblate (Spanish)

Currently translated at 88.8% (8 of 9 strings)

Translation: product-attribute-15.0/product-attribute-15.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-15-0/product-attribute-15-0-product_net_weight/es/

Commit:
[MIG] product_category_active: Migration to 16.0

[UPD] Update product_category_active.pot

[UPD] README.rst

product_category_active 16.0.1.0.1

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_category_active/

Commit:
[FW][FIX]product_sequence: remove dependence

[UPD] README.rst

product_sequence 16.0.2.0.0

Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/it/

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (16 of 16 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/pt_BR/

Translated using Weblate (Spanish)

Currently translated at 100.0% (16 of 16 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/es/

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/

RabbitJon-S73 committed:
[MIG] product_net_weight: Migration to 16.0

sergio-teruel committed:
[IMP] product_net_weight: Adapt module like Odoo manage weight field with multiple variants

lmignon committed:
[FIX] product_logistics_uom: Store value in system UOM

Before this change, the volume and weight values on product were no more stored into the expected kg and m3 uoms. To know the effective value you had to convert the value using the specified uom on the record. As side effect the arithmetic operations done on these fields in others addon as for exemple in 'delivery' were not correct.

This change restore the default behavior by always storing the values into the default uoms whatever the uom specified on the product. 2 new fields are added on the product to allows the user to see and managed these values in the specified uom. When values are updated through an update of these new fields, the new values are converted into the default uoms and the result stored into the original fields.

fixes #1312

PicchiSeba committed:
Added translation using Weblate (Italian)

PicchiSeba committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (13 of 13 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_logistics_uom/it/

PicchiSeba committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_category_active/it/

[UPD] README.rst

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (13 of 13 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_logistics_uom/it/

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_net_weight/

Commit:
Add product_set

[UPD] Update product_set.pot

[UPD] README.rst

Translated using Weblate (Spanish)

Currently translated at 100.0% (23 of 23 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/es/

[UPD] README.rst

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (23 of 23 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/pt_BR/

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/

Translated using Weblate (Italian)

Currently translated at 100.0% (23 of 23 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (28 of 28 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_logistics_uom/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_net_weight/es/

OCA-git-bot committed:
[UPD] README.rst

rousseldenis committed:
[IMP] product_packaging_level: Improve README

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
oca-github-bot post-merge updates

rousseldenis committed:
[FIX] product_packaging_level: Use correct field in hook

OCA-git-bot committed:
oca-github-bot post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (25 of 25 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

JordiBForgeFlow committed:
[FIX] product_packaging_dimension: post-migrate script should only apply query if column max_weight exists.
This column is no longer defined in 16.0

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_dimension
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_dimension/

anothingguy committed:
[IMP] product_packaging_level: configure product packaging name

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (32 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

josep-tecnativa committed:
[IMP] product_category_active: test performance improvement
- Include context keys for avoiding mail operations overhead.

[BOT] post-merge updates

josep-tecnativa committed:
[IMP] product_sequence: test performance improvement
- Switch to setUpClass for avoiding repeat the same setup for each test.
- Include context keys for avoiding mail operations overhead.

LoisRForgeFlow committed:
[ADD] product_pricelist_fixed_currency_rate

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (28 of 28 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_logistics_uom/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_net_weight
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_net_weight/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 71.8% (23 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (32 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

OCA-git-bot committed:
[BOT] post-merge updates

Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_sequence
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_sequence/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_category_active/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (28 of 28 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_logistics_uom/it/

psc committed:
Translated using Weblate (German)

Currently translated at 84.6% (44 of 52 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_medical/de/

Commit:
[UPD] Update product_pricelist_fixed_currency_rate.pot

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_pricelist_fixed_currency_rate
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_pricelist_fixed_currency_rate/it/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (12 of 12 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_pricelist_fixed_currency_rate
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_pricelist_fixed_currency_rate/es/

Commit:
[IMP] product_set: Merge with sale_product_set_layout

[UPD] Update product_set.pot

[BOT] post-merge updates

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/

Translated using Weblate (Spanish)

Currently translated at 100.0% (29 of 29 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/es/

Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/it/

NICO-SOLUTIONS committed:
[IMP] product_sequence: pre-commit auto fixes

NICO-SOLUTIONS committed:
[MIG] product_sequence: Migration to 17.0

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (52 of 52 strings)

Translation: product-attribute-14.0/product-attribute-14.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-14-0/product-attribute-14-0-product_medical/es/

Commit:
[16.0][FIX] product_packaging_level: Preserve sequence order

JuMiSanAr committed:
Add product_packaging_type_vendor

Commit:
[IMP]product_packaging_level: recover name_get policy flow

Commit:
[UPD] Update product_packaging_type_vendor.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update product_sequence.pot

OCA-git-bot committed:
[BOT] post-merge updates

pilarvargas-tecnativa committed:
[IMP] product_set: Move the base logic of the transient model defined in the module sale_product_set.

The reason to move this logic is that there are other modules that extend
product_set as for example stock_product_set but at the same time make use
of the transient model to define a wizard. For this reason it is better to
have the logic available in this base module and avoid duplicating code or
inheriting from sale_product_set and what this implies in their respective
dependencies with the only need to extend the transient model.
In addition, the transient model is renamed to make it clearer to identify
that it is this type of model.

TT48100

[UPD] Update product_set.pot

[BOT] post-merge updates

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/

Translated using Weblate (Italian)

Currently translated at 100.0% (32 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_set/it/

NICO-SOLUTIONS committed:
[IMP] product_set: pre-commit auto fixes

NICO-SOLUTIONS committed:
[MIG] product_set: Migration to 17.0

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Commit:
[UPD] Update product_set.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-17.0/product-attribute-17.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_set/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (32 of 32 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (34 of 34 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (33 of 33 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_set/it/

NICO-SOLUTIONS committed:
Translated using Weblate (German)

Currently translated at 100.0% (33 of 33 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_set/de/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (33 of 33 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_set/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (34 of 34 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

pedrobaeza committed:
[FIX] product_set: pre-commit

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

bizzappdev committed:
[IMP] product_logistics_uom: pre-commit auto fixes

bizzappdev committed:
[MIG] product_logistics_uom: Migration to 17.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (34 of 34 strings)

Translation: product-attribute-16.0/product-attribute-16.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_packaging_level/it/

nguyenminhchien committed:
[IMP] product_packaging_dimension: pre-commit auto fixes

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (33 of 33 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_set/sv/

bizzappdev committed:
[IMP] product_packaging_level: pre-commit auto fixes

bizzappdev committed:
[MIG] product_packaging_level: Migration to 17.0

remihb committed:
[IMP] product_net_weight: pre-commit auto fixes

remihb committed:
[MIG] product_net_weight: Migration to 17.0

nguyenminhchien committed:
[MIG] product_packaging_dimension: Migration to 17.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_logistics_uom/it/

kobros-tech committed:
[MIG] product_pricelist_fixed_currency_rate: Migration to 17.0

Commit:
[UPD] Update product_packaging_dimension.pot

OCA-git-bot committed:
[BOT] post-merge updates

WildfootW committed:
Translated using Weblate (Chinese (Traditional) (zh_TW))

Currently translated at 93.9% (31 of 33 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_set/zh_TW/

uvid-alen committed:
Added translation using Weblate (Croatian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (33 of 33 strings)

Translation: product-attribute-17.0/product-attribute-17.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-17-0/product-attribute-17-0-product_set/it/

rf-ife committed:
[MIG] product_net_weight: adjust form layout

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[BOT] post-merge updates

LucasTran380381 committed:
[IMP] product_packaging_dimension: pre-commit auto fixes

LucasTran380381 committed:
[MIG] product_packaging_dimension: Migration to 18.0

LucasTran380381 committed:
[IMP] product_logistics_uom: pre-commit auto fixes

LucasTran380381 committed:
[MIG] partner_logistics_uom: Migration to 18.0

Commit:
[IMP] product_medical: pre-commit auto fixes

Commit:
[MIG] product_medical: Migration to 18.0

sbejaoui committed:
[18.0][MIG] product_set

Commit:
[UPD] Update product_pricelist_fixed_currency_rate.pot

OCA-git-bot committed:
[BOT] post-merge updates

chrisb-c01 committed:
[MIG] product_sequence: Migration to 18.0

alexis-via committed:
[IMP] product_net_weight: pre-commit auto fixes for v18 migration

alexis-via committed:
[MIG] product_net_weight to v18

BhaveshHeliconia committed:
[MIG] product_pricelist_fixed_currency_rate: Migration to 18.0

trisdoan committed:
[IMP] product_packaging_level: pre-commit auto fixes

trisdoan committed:
[MIG] product_packaging_level: Migration to 18.0

trisdoan committed:
[FIX] product_packaging_level: cannot input name with user_defined policy

victoralmau committed:
[IMP] product_category_active: pre-commit auto fixes

victoralmau committed:
[MIG] product_category_active: Migration to 17.0

TT52280

thienvh332 committed:
[IMP] product_packaging_type_vendor: pre-commit auto fixes

thienvh332 committed:
[MIG] product_packaging_type_vendor: Migration to 18.0 and renaming to product_packaging_level_vendor

LucasTran380381 commented:
![image](https://github.com/user-attachments/assets/a9cf0d95-7bcd-44e1-b10d-ce8d69fb69a3)
LGTM

Commit:
[UPD] Update product_category_active.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1751 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update product_logistics_uom.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/

OCA-git-bot committed:
Merge PR #1793 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update product_pricelist_fixed_currency_rate.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_pricelist_fixed_currency_rate
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_pricelist_fixed_currency_rate/

OCA-git-bot committed:
Merge PR #1753 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_medical.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1758 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_set.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_medical/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_set
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_set/

BhaveshHeliconia committed:
[MIG] product_category_active: Migration to 18.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (51 of 51 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_medical
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_medical/it/

OCA-git-bot committed:
Merge PR #1787 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update product_logistics_uom.pot

Commit:
[UPD] Update product_net_weight.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (31 of 31 strings)

Translation: product-attribute-18.0/product-attribute-18.0-product_logistics_uom
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_logistics_uom/it/

OCA-git-bot committed:
Merge PR #1752 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update product_packaging_dimension.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1780 into 18.0

Signed-off-by StefanRijnhart

Commit:
[UPD] Update product_sequence.pot

OCA-git-bot committed:
[BOT] post-merge updates

legalsylvain committed:
[FIX] product_category_active: harmonization of category display.
In odoo, archived items are not displayed by default. (see users view, products, etc...)
The product_category_active module doesn't follow this convention because it is changing the context of the action of product.category. As a result, all categories are displayed, polluting visibility with obsolete items.
- Remove the active_test in the context of the action.
- create a migration script to remove the active_test on existing instances.
- add filters to display archived and 'archived or active' categories, in case you want to see all categories

OCA-git-bot committed:
Merge PR #1806 into 18.0

Signed-off-by sebalix

Commit:
[UPD] Update product_packaging_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_packaging_level
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_packaging_level/

OCA-git-bot committed:
Merge PR #1851 into 18.0

Signed-off-by legalsylvain

Commit:
[UPD] Update product_category_active.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-18.0/product-attribute-18.0-product_category_active
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-18-0/product-attribute-18-0-product_category_active/
